aten/src/ATen/native/cuda/UpSample.cuh

```diff
@@ -158,51 +158,42 @@ __device__ __forceinline__ static int nearest_neighbor_compute_source_index(
     const float scale,
     int dst_index,
     int input_size) {
-  const int src_index = min<int>(
-      static_cast<int>(floorf(dst_index * scale)), input_size - 1);
+  const int src_index =
+      min<int>(static_cast<int>(floorf(dst_index * scale)), input_size - 1);
   return src_index;
 }
 
-/* just affect UpSampleBicubic2d.cu */
-/* TODO: change width and height order in the arguments */
-/* TODO: maybe change x and y order in the arguments */
-/* TODO: maybe change channel and batch order in the arguments */
+/* Used by UpSampleBicubic2d.cu */
 template <typename scalar_t>
 __device__ __forceinline__ static scalar_t upsample_get_value_bounded(
     const PackedTensorAccessor<scalar_t, 4>& data,
-    int channel,
     int batch,
-    int width,
+    int channel,
     int height,
-    int x,
-    int y) {
-  int access_x =
-      max<int>(min<int>(x, width - 1), static_cast<int>(0));
-  int access_y =
-      max<int>(min<int>(y, height - 1), static_cast<int>(0));
+    int width,
+    int y,
+    int x) {
+  int access_x = max<int>(min<int>(x, width - 1), static_cast<int>(0));
+  int access_y = max<int>(min<int>(y, height - 1), static_cast<int>(0));
```

Comment:
If you change the argument order to match the indexing order in `data[][][][]`, I'd also put `int access_y ...` above `int access_x ...`.  That makes it absolutely clear that the order is intentional.

And please remove the cast in `static_cast<int>(0)`.  An integer literal without a suffix is already an `int` if it fits the integer size.

Reply:
you're right. I will be better change the order here too. thanks!
sure! I will remove it .. probably it was done by my script ... also I will check the script and remove this rule. thanks!

Reply:
I recall now, before it was `static_cast<int_64>(0)` and I changed all index from `int64_t` to `int` .. and I forgot to fix these cast. thanks!